CardiorespiratoryFitness/CardiorespiratoryFitness/Model/CRFTaskInfo.swift

```diff
@@ -36,17 +36,35 @@ import Foundation
 /// A list of all the tasks included in this module.
 public enum CRFTaskIdentifier : String, Codable, CaseIterable {
     
+    /// Training task for measuring your heart rate.
     case training = "Heartrate Training"
     
+    /// Measure your heart rate while resting.
     case resting = "Resting Heartrate"
     
+    /// Measure your heart rate, first thing in the morning before getting out of bed.
+    case restingMorning = "Morning Heartrate"
```

Comment:
After lots of discussion early on about what is the gold standard for how to take a resting heart rate, consensus was that we should mimic what happens in the clinic, which is just sitting and resting.  Similarly, that is really the only thing where are explicitly asking for that context (as opposed to having a reminder for first thing in the morning, etc).  I think we do not need this fork of the task.

Reply:
@DanWebster 

How did you get the back button to appear there? Is that in simulator or on device? There isn't suppose to be a back button.

Reply:
@syoung-smallwisdom  This was in simulator on SE, but is only seen in the 'morning heart rate' (now commented out) path.  Not there in the training/resting, just a single button to progress forward